### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by lordfriend**
Add unit test for pg-store, normalize.
using alsatian as unit test framework and runner.

### CODE REVIEW

**Comment by acgotaku** [comment]
Maybe we need to add `BANGUMI_MOE ` test case.

File: src/types.ts [hunk 1]
```diff
@@ -18,4 +19,14 @@ export interface PersistentStorage<T> {
 export interface Scraper {
     start(): Promise<any>;
     end(): Promise<any>;
-}
\ No newline at end of file
+}
+
+export interface ConfigLoader {
```
Comment (acgotaku) [inline src/types.ts]: Probably use `class` is better. https://stackoverflow.com/questions/51716808/when-use-a-interface-or-class-in-typescript

Comment (lordfriend) [reply src/types.ts]: We can use class for IoC binding, but if you want to have two different ConfigLoader implementation for DI, then interface is better.
class is supported: https://github.com/inversify/InversifyJS/blob/master/wiki/classes_as_id.md
but we need to sure there is no need for a different implementation for this type.

**Comment by lordfriend** [comment]
> Maybe we need to add `BANGUMI_MOE ` test case.

You min the test for pg-store? 
I haven't added a unit-test for scrapers. because I don't know how to test it without relying on a third party service.

**Pull request merged**